versions.tf

```
@@ -0,0 +1,10 @@
+# Terraform and Provider versions
+terraform {
+  required_version = ">= 1.2.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.9.0"
```

This is a cleaner requirement

```suggestion
      version = ">= 4.9"
```

---

Removing the patch version from the constraint has led to errors in PRs generated by renovatebot (specifically around the explicit constraints which are generated for the terraform.lock.hcl file). We'll keep it as-is for now, and hopefully they'll fix renovate/aws provider version resolution (or provide a work-around) in the future.